src/java/voldemort/store/readonly/swapper/AdminStoreSwapper.java

```diff
@@ -223,10 +224,14 @@ private String fetch(String storeDir) {
             } catch(ExecutionException e) {
                 fetchErrors = true;
                 if(e.getCause() instanceof QuotaExceededException) {
-                    numQuotaExceededException++;
+                    //numQuotaExceededException++;
+                    QuotaExceededExceptions.add(node.getId());
                     fetchResponseMap.put(node, new Response((QuotaExceededException) e.getCause()));
                 } else if(e.getCause() instanceof InvalidBootstrapURLException) {
```

Comment:
When a store is failed to be created, shoudln't it return a meaningful exception instead of sending InvalidBootstrapUrl exceptions ?